cobalt/h5vcc/h5vcc_accessibility.cc

```diff
@@ -74,11 +78,23 @@ void H5vccAccessibility::set_built_in_screen_reader(bool value) {
 bool H5vccAccessibility::high_contrast_text() const {
   SbAccessibilityDisplaySettings settings;
   memset(&settings, 0, sizeof(settings));
-
+#if SB_API_VERSION >= 16
+  auto accessibility_api = static_cast<const CobaltExtensionAccessibilityAPI*>(
+      SbSystemGetExtension(kCobaltExtensionAccessibilityName));
+  if (accessibility_api &&
+      strcmp(accessibility_api->name, kCobaltExtensionAccessibilityName) == 0 &&
+      accessibility_api->version >= 1) {
+    if (!accessibility_api->GetDisplaySettings(&settings)) {
+      return false;
+    }
+  } else {
+    return false;
+  }
+#else   // SB_API_VERSION >= 16
```

Comment:
nit: looks like an extra space here. Can you please check the rest of the PR for this, too?

Reply:
There are three spaces but I cannot change it since it was formatted by pre-commit script. The script puts three spaces for "#else  //  SB_API_VERSION >= 16" line.